src/brevitas_examples/stable_diffusion/main.py

```diff
@@ -360,29 +383,29 @@ def input_zp_stats_type():
         if args.quantize_sdp:
             assert args.share_qkv_quant, "Currently SDPA quantization is supported only with shared QKV quantization"
             # TODO: reformat this
-            float_sdpa_quantizers = generate_quantizers(
+            sdpa_quantizers = generate_quantizers(
                 dtype=dtype,
                 device=args.device,
-                weight_bit_width=weight_bit_width,
-                weight_quant_format='float_ocp_e4m3',
-                weight_quant_type='sym',
-                weight_param_method=args.weight_param_method,
-                weight_scale_precision=args.weight_scale_precision,
-                weight_quant_granularity=args.weight_quant_granularity,
-                weight_group_size=args.weight_group_size,
-                quantize_weight_zero_point=args.quantize_weight_zero_point,
-                quantize_input_zero_point=args.quantize_input_zero_point,
-                input_bit_width=args.linear_output_bit_width,
-                input_quant_format='float_ocp_e4m3',
-                input_scale_type=args.input_scale_type,
-                input_scale_precision=args.input_scale_precision,
-                input_param_method=args.input_param_method,
-                input_quant_type='sym',
-                input_quant_granularity=args.input_quant_granularity,
-                input_kwargs=input_kwargs)
+                weight_bit_width=args.sdpa_bit_width,
+                weight_quant_format=args.sdpa_quant_format,
+                weight_quant_type=args.sdpa_quant_type,
+                weight_param_method=args.sdpa_param_method,
+                weight_scale_precision=args.sdpa_scale_precision,
+                weight_quant_granularity=args.sdpa_quant_granularity,
```

Comment:
@Giuseppe5, interested in your thoughts on this. This weight quantizer is used for convolutions if `args.override_conv_quant_config=True`. Current version prevents us from doing channelwise scaling here...